cmd/snapshot-controller/main.go

```diff
@@ -48,6 +50,9 @@ var (
 
 	leaderElection          = flag.Bool("leader-election", false, "Enables leader election.")
 	leaderElectionNamespace = flag.String("leader-election-namespace", "", "The namespace where the leader election resource exists. Defaults to the pod namespace if not set.")
+
+	httpEndpoint = flag.String("http-endpoint", "", "The TCP network address where the http server will listen (example: `:8080`). The default is empty string, which means the http endpoint is disabled.")
```

Comment:
nit: could be more descriptive on what the server is, i.e:

"The TCP network address where the HTTP server for diagnostics, including metrics, will listen (example: `:8080`). The default is empty string, which means the server is disabled."

Reply:
Thanks, added this in